.pylintrc

```diff
@@ -280,7 +280,9 @@ function-naming-style=snake_case
 #function-rgx=
 
 # Good variable names which should always be accepted, separated by a comma.
-good-names=i,
+good-names=dict,
```

Comment:
I agree with @boecklic we should not use dict as a variable name as it can confuse the reader with the python standard dict, in fact using `dct` instead of `dict` is accepted by pylint !

Reply:
I took this configuration either from an existing project or from the doc guidelines about it. 

That being said, I'm clearly not using dict as a variable, and it will be removed from the allowed list.